<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtRemote</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/logo.ico</normaloff>../img/logo.ico</iconset>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QtRemote&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
     color: lightgray;
     background-color: transparent;
}
QMainWindow {
    border-image: url('img/background.gif') 0 0 0 0 stretch;
    background-repeat: no-repeat;
    min-width: 1121;
    min-height:766;
    padding: 10px;

}
QGroupBox {
    border: 2px dashed gray;
    border-radius: 3px;
    padding: 5px;
}
QPushButton {
    background-color: lightblue;
    border-radius: 5px;
    margin: 5px;
    padding: 5px;
    color: white;
}
QLabel {
    border-radius: 2px;
    margin-left: 5px;
    margin-right: 5px;
    padding-left: 5px;
    padding-right: 5px;
}
QLineEdit {
    border: 2px solid gray;
    border-radius: 2px;
    margin-left: 5px;
    margin-right: 5px;
    padding-left: 2px;
    padding-right: 2px;
}
QTableWidget {
    border-radius: 5px;
    padding: 7px;
}
QHeaderView::section {
	border-bottom: 2px solid gray;
    background-color: transparent;
}
QAxWidget {
    /*max-width: 192;
    max-height: 108;*/
}
QPlainTextEdit {
    border: 2px solid gray;
    border-radius: 2px;

}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>700</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QGroupBox" name="guoup_box_add_topdesk">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>新增桌面</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_address">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>地址</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::AutoText</enum>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="edit_address">
                 <property name="toolTip">
                  <string>物理机ip地址</string>
                 </property>
                 <property name="statusTip">
                  <string>物理机ip地址</string>
                 </property>
                 <property name="text">
                  <string extracomment="主机名称"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_account">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>用户名</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::AutoText</enum>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="edit_account">
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="statusTip">
                  <string/>
                 </property>
                 <property name="text">
                  <string extracomment="主机名称"/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_password">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>密码</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::AutoText</enum>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="edit_password">
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="statusTip">
                  <string/>
                 </property>
                 <property name="text">
                  <string extracomment="主机名称"/>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_domain">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>  域名</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::AutoText</enum>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="edit_domain">
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="statusTip">
                  <string/>
                 </property>
                 <property name="text">
                  <string>LQAD</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="button_append">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>40</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>添加</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="group_box_dingding">
          <property name="title">
           <string>钉钉通知</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_webhook">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>Webhook</string>
               </property>
               <property name="textFormat">
                <enum>Qt::AutoText</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edit_webhook">
               <property name="toolTip">
                <string>钉钉机器人Webhook</string>
               </property>
               <property name="statusTip">
                <string>钉钉机器人Webhook</string>
               </property>
               <property name="text">
                <string extracomment="主机名称">https://oapi.dingtalk.com/robot/send?access_token=c3f39994904245f2d73847648aa17a8c4404654da1c2dd1b21ee52f2316fa55f</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Normal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_secret">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string> Secret</string>
               </property>
               <property name="textFormat">
                <enum>Qt::AutoText</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edit_secret">
               <property name="toolTip">
                <string>钉钉机器人Secret</string>
               </property>
               <property name="statusTip">
                <string>钉钉机器人Secret</string>
               </property>
               <property name="text">
                <string>SEC1187c5c3e7a313a63169265a1e7c6b14bcdd93739dad1c6cf64dfa000be31697</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Normal</enum>
               </property>
               <property name="cursorMoveStyle">
                <enum>Qt::LogicalMoveStyle</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_per">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>每</string>
               </property>
               <property name="textFormat">
                <enum>Qt::AutoText</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edit_per">
               <property name="maximumSize">
                <size>
                 <width>38</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string/>
               </property>
               <property name="statusTip">
                <string/>
               </property>
               <property name="text">
                <string>30</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_check">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>分钟检查一次</string>
               </property>
               <property name="textFormat">
                <enum>Qt::AutoText</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QGroupBox" name="group_box_desk">
          <property name="title">
           <string>桌面列表</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QTableWidget" name="table_desk">
             <property name="minimumSize">
              <size>
               <width>700</width>
               <height>0</height>
              </size>
             </property>
             <property name="textElideMode">
              <enum>Qt::ElideMiddle</enum>
             </property>
             <property name="gridStyle">
              <enum>Qt::SolidLine</enum>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>地址</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>用户名</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>密码</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>域名</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>状态</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>桌面</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="group_box_log">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>日志</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPlainTextEdit" name="plain_text_log">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="maximumBlockCount">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>700</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
